doc/requirements.md

```diff
@@ -0,0 +1,20 @@
+## Technology Stack
+
+### Main dependencies
+ 
+* Python - Implementation language
+* HDF5 - Used for storing and managing data
+* numpy and scipy - Fundamental packages for scientific computing with Python
+
+### Binary dependencies
+
+Software  | Version(s)
+--------- | ----------
+Python | 2.7, 3
```

Comment:
I would keep Python 2.7 Officially Python 3 is not supported even if unofficially it works.